sematic/ui/packages/common/src/pages/RunDetails/externalResource/ExternalResource.tsx

```diff
@@ -0,0 +1,79 @@
+import Timeline from "@mui/lab/Timeline";
+import timelineItemClasses from "@mui/lab/TimelineItem/timelineItemClasses";
+import Alert from "@mui/material/Alert";
+import { styled } from "@mui/system";
+import { useMemo, useEffect } from "react";
+import { ExternalResourceHistorySerialization, Run } from "src/Models";
+import { useExternalResource } from "src/hooks/externalResourceHooks";
+import ExternalResourceState from "src/pages/RunDetails/externalResource/ExternalResourceState";
+
+const ThinTimeline = styled(Timeline)`
```

Comment:
Can we get a little more top margin? It seems a little too close to the top.

Reply:
Sure, like this:

<img width="1132" alt="image" src="https://github.com/sematic-ai/sematic/assets/133257643/ac9a70f3-4790-4f55-bbc9-6330cba962eb">